vmdb/app/models/miq_server/server_smart_proxy.rb

```diff
@@ -116,7 +116,7 @@ def scan_sync_vm(ost)
         temp_args['ems'][:use_vim_broker]      = MiqServer.use_broker_for_embedded_proxy?(temp_args['ems']['connect_to'])
         temp_args['ems'][:vim_broker_drb_port] = MiqVimBrokerWorker.drb_port if temp_args['ems'][:use_vim_broker]
         # RHEV-M setup
-        if v.vendor.to_s == 'RedHat'
+        if v.vendor.to_s == 'Red Hat'
```

Comment:
ugh...I hate this code.  Is it possible to just use kind_of? or some other kind of check.

Yeah, I know you are just fixing the brand issues....just thinking ahead